data/products/chost/sle15/packages.yaml

```diff
@@ -18,12 +18,13 @@ packages:
       - nfs-client
       - open-iscsi
       - pciutils
+      - rsyslog
       - runc
       - socat
       - supportutils-plugin-suse-public-cloud
       - tcpdump
       - vim
-      - vlan
```

Comment:
Please split the vlan removal into a separate PR. What replaces vlan, how does k8s setup networking and does SAP (Gardener) use anything form the vlan package are questions we need to answer first and I think we should not block the other changes while those questions get answered.

Reply:
Can do, but not sure it's worth it. Quoting the vlan website (http://www.candelatech.com/~greear/vlan.html, last change 2005):

> 802.1Q VLAN code is now part of the official kernel, and has been for years and years. MAC-VLAN code has been added since around 2.6.29. It is very unlikely that you need to download anything from this site, the packages are left here for posterity's sake.
> 802.1Q VLANs and MAC-VLANs can be created with the 'ip' utility (vconfig works for vlans, but is crufty and deprecated).

Reply:
OK, if it is part of the kernel and the kernel also delivers the user space tools, please double check, then we are OK with this drop.

Reply:
The `vlan` package contains the `vconfig` user space tool which is about as deprecated as `ifconfig`.

Reply:
I understand that. The question is if you are willing to cause production issues, and deal with the inevitable escalations, at SAP because vconfig is obsolete and gardner may or may not be using it? If you are willing to go down that route without checking then I am OK with merging this. From my point of view it is better to ask and then follow up with a separate PR after we know that we will not break anything.

Reply:
Sure, that's why I dropped the removal from the PR.